lib/util/trip/ride_card.dart

```diff
@@ -145,23 +133,75 @@ class _RideCardState extends TripCardState<RideCard> {
     }
   }
 
-  Color pickColor() {
-    return _ride!.status.isCancelled() || _ride!.status == RideStatus.rejected
-        ? Theme.of(context).disabledColor.withOpacity(0.05)
-        : Theme.of(context).cardColor;
+  Color pickBannerColor() {
+    switch (_ride!.status) {
+      case RideStatus.preview:
+        return Theme.of(context).cardColor;
+      case RideStatus.pending:
+        return Theme.of(context).disabledColor;
+      case RideStatus.approved:
+        return Theme.of(context).own().success;
+      case RideStatus.rejected:
+        return Theme.of(context).errorColor;
```

Comment:
Somehow `errorColor` is different than `colorScheme.error`. No idea why, but we always used the latter because it is more accurate. So please change that for your PR